overlay/overlay_launchers.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2005-2016 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_OVERLAY_LAUNCHERS_H_
+#define MUMBLE_OVERLAY_LAUNCHERS_H_
+
+static const char *overlayLaunchers[] = {
+	"Steam.exe", // Steam
+	"EALaunchHelper.exe", // Origin
```

Comment:
With Battlefield 4 it is _BF4WebHelper.exe_. I guess that every game has got its own executable.
I'm going to try with Battlefield 1 once it finishes downloading the update.